### PR TITLE
Improve client UI

### LINKED ISSUE #1695: Improve UI/UX
The client uses two types of buttons. Normal ones (grey) and important ones (white, e.g. "create game" or "search"). White is not really noticable. The white buttons should have a bright color to make them more visible. The button color needs to fit with the rest of the client coloring.
Also the "more" buttons in the vault are not where you expect them. They are on the far right and on the same height as the category headings, making it confusing what the right button is.

### PR DESCRIPTION
**Pull request by BlackYps**
WIP
Fixes #1695 

The white buttons are now orange to make them stand out more. The ugly green is replaced by shades of blue (complementary color).

grafik

The more buttons in the map vault are now located directly after the last map card.
grafik

If you have concerns or other/further ideas to enhance the UI please let me know. More input helps me to find better solutions.

### CODE REVIEW

File: src/main/java/com/faforever/client/map/MapVaultController.java [hunk 1]
```diff
@@ -353,6 +358,25 @@ private void replaceSearchResult(List<MapBean> maps, Pane pane) {
...
+          break;
+        case "mostPlayedPane":
+          pane.getChildren().add(moreMostPlayedButton);
+          break;
+        case "ladderPane":
+          pane.getChildren().add(moreLadderButton);
+          break;
+        default:
+          // Do nothing
+      }
```
Comment (1-alex98) [inline src/main/java/com/faforever/client/map/MapVaultController.java:379]: What's that for UI is supposed to be edited in the fxml

File: src/main/java/com/faforever/client/main/event/NavigationItem.java [hunk 2]
```diff
@@ -10,7 +10,11 @@
   LEADERBOARD("theme/leaderboard/leaderboards.fxml"),
   UNITS("theme/units.fxml"),
   TUTORIALS("theme/tutorial.fxml"),
-  TOURNAMENTS("theme/tournaments/tournaments.fxml");
+  TOURNAMENTS("theme/tournaments/tournaments.fxml"),
+
+  CUSTOM_GAMES("theme/play/custom_games.fxml"),
+  RANKED_1V1("theme/play/ranked_1v1.fxml"),
+  COOP("theme/play/coop/coop.fxml");
```
Comment (1-alex98) [inline src/main/java/com/faforever/client/main/event/NavigationItem.java]: Do u know what u are doing here?

File: src/main/java/com/faforever/client/mod/ModVaultController.java [hunk 3]
```diff
@@ -289,6 +293,19 @@ private void appendSearchResult(List<ModVersion> modVersions, Pane pane) {
...
+          break;
+        case "newestPane":
+          pane.getChildren().add(moreNewestButton);
+          break;
+        case "highestRatedPane":
+          pane.getChildren().add(moreHighestRatedButton);
+          break;
+        default:
+          // Do nothing
+      }
```
Comment (1-alex98) [inline src/main/java/com/faforever/client/mod/ModVaultController.java:308]: Same here

File: src/main/java/com/faforever/client/play/PlayController.java [hunk 4]
```diff
@@ -51,7 +63,7 @@ public void initialize() {
       } else if (newValue == coopTab) {
         eventBus.post(new OpenCoopEvent());
       }
-    });
+    });*/
```
Comment (1-alex98) [inline src/main/java/com/faforever/client/play/PlayController.java]: No commented out code... even if u did not add it

File: src/main/resources/theme/style-webview.css [hunk 5]
```diff
@@ -16,7 +16,7 @@ html, body {
 
 a {
   /* Use same as -fx-accent */
-  color: #009688;
+  color: #3F51B5;
```
Comment (1-alex98) [inline src/main/resources/theme/style-webview.css:19]: We should make this a variable

**Review by 1-alex98 — CHANGES_REQUESTED** [review]

**Comment by 1-alex98** [comment]
And tests are failing

**Comment by 1-alex98** [comment]
Also we could try to make a theme first and look how ppl like it

File: src/main/java/com/faforever/client/map/MapVaultController.java (hunk 1 above)
Comment (BlackYps) [reply src/main/java/com/faforever/client/map/MapVaultController.java:379]: This adds the more buttons into the flowpane right after the flowpane has been populated with maps. I don't know if it is possible to do this in fxml while making sure that the button always is the last element.

File: src/main/java/com/faforever/client/main/event/NavigationItem.java (hunk 2 above)
Comment (BlackYps) [reply src/main/java/com/faforever/client/main/event/NavigationItem.java]: No :P
I'm currently trying to get my head around this eventhandler stuff. It's not supposed to stay like that.

File: src/main/java/com/faforever/client/play/PlayController.java (hunk 4 above)
Comment (BlackYps) [reply src/main/java/com/faforever/client/play/PlayController.java]: Yes, I just commited the current status. The whole file is pretty broken right now.

File: src/main/resources/theme/style-webview.css (hunk 5 above)
Comment (BlackYps) [reply src/main/resources/theme/style-webview.css:19]: I agree, but this is a different css file. I don't know if that is possible.

File: src/main/java/com/faforever/client/play/PlayController.java [hunk 6]
```diff
@@ -61,15 +76,21 @@ protected void onDisplay(NavigateEvent navigateEvent) {
     try {
       if (Objects.equals(navigateEvent.getClass(), NavigateEvent.class)
           || navigateEvent instanceof OpenCustomGamesEvent) {
-        playRootTabPane.getSelectionModel().select(customGamesTab);
+        Node node = customGamesController.getRoot();
```
Comment (BlackYps) [inline src/main/java/com/faforever/client/play/PlayController.java]: I get an exception on this line. Any idea what the problem is? Is the customGamesController not initialied yet?

**Comment by BlackYps** [comment]
continued in #1699

**Pull request closed without merging**